### DISCUSSION TITLE
Comments: What I've learnt reading Jason Miller's source code

### DISCUSSION THREAD

**Discussion by jiayihu** [jiayihu/blog#60] [Announcements]
This issue is reserved for comments to [What I've learnt reading Jason Miller's source code](https://blog.jiayihu.net/what-i-ve-learnt-reading-jason-miller-s-source-code/). Leave a comment below and it will be shown in the blog page.

**Comment by MaxArt2501** [comment]
As I said to you privately, this is a very nice article. Keep it up 👍 

All those tricks in the code are nice. It would be excellent if they had some related comments why those solutions were adopted. I think it's especially useful for open source projects.
